app/delivery/send_to_providers.py

```diff
@@ -152,24 +146,6 @@ def send_email_to_provider(notification: Notification):
         for key in file_keys:
             check_file_url(personalisation_data[key]["document"], notification.id)
             sending_method = personalisation_data[key]["document"].get("sending_method")
-            # Check if a MLWR sid exists
-            if (
```

Comment:
this is correct that we needed to remove this code?

Reply:
Yes, Jimmy removed this as part of removing references to `assemblyline`.